tdrs-backend/tdpservice/parsers/schema_defs/tanf/t5.py

```diff
@@ -0,0 +1,81 @@
+"""Schema for HEADER row of all submission types."""
+
+
+from ...util import SchemaManager
+from ...fields import EncryptedField, Field, tanf_ssn_decryption_func
+from ...row_schema import RowSchema
+from ... import validators
+from tdpservice.search_indexes.models.tanf import TANF_T5
+
+
+t5 = SchemaManager(
+      schemas=[
+        RowSchema(
+          model=TANF_T5,
+          preparsing_validators=[
+              validators.hasLength(71),
+          ],
+          postparsing_validators=[],
+          fields=[
+              Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
+                    required=True, validators=[]),
+              Field(item="4", name='RPT_MONTH_YEAR', type='number', startIndex=2, endIndex=8,
```

Comment:
same as above. i dont feel strongly about it, but it is a date field and will eventually need a date to calculate AGE in validation ticket.